Makefile

```diff
@@ -260,7 +260,7 @@ SOURCE_FILES = \
   IROperator.cpp \
   IRPrinter.cpp \
   IRVisitor.cpp \
-  JITCompiledModule.cpp \
+  JITModule.cpp \
```

Comment:
Probably need to update CMakeLists with the JITModule.* and the new runtime modules.